GTG/gtk/browser/main_window.py

```diff
@@ -297,16 +297,32 @@ def _init_about_dialog(self):
         """
         Show the about dialog
         """
+        # These lines should be in info.py, but due to their dynamic nature
+        # there'd be no way to show them translated in Gtk's About dialog:
+
+        translated_copyright = _("Copyright © 2008-%d the GTG contributors.") % datetime.date.today().year
+
+        UNITED_AUTHORS_OF_GTGETTON = [_("many contributors around the world."), # GTK prefixes this intro line with "Created by "
+            _("The GTG project is maintained/administered by:"),
+            info.AUTHORS_MAINTAINERS,
+            _("This release was brought to you by the efforts of these people:"),
+            info.AUTHORS_RELEASE_CONTRIBUTORS,
+            # FIXME: this string refuses to translate the variables, no matter if I use %(foo)s or {foo} % {"foo": "bar"}
```

Comment:
Then does it at least work with what you have now? This looks to be a f-string without it being marked as such, and I'm not sure how it'd work like this. Testing in python interpreter shows it won't work.
I think `.format(...)` works with `{var}`, but `%` doesn't. If you want to keep a readable name, and %(var)s doesn't work right, and you don't want to just use two `%s` with translator comment, then maybe like

```
            _("Many others contributed to GTG over the years.\nYou can see them on {OH_stats} and {GH_stats}.".format(
                    OH_stats='<a href="https://www.openhub.net/p/gtg/contributors">OpenHub</a>',
                    GH_stats='<a href="https://github.com/getting-things-gnome/gtg/graphs/contributors">GitHub</a>'),
```

Reply:
I had tried the .format trick too, with the python-brace-format thing, and had no luck with that particular line, hence the FIXME. The rest of the portions of the translatable string (and the ones above) work, so I thought OK, I can live with that, it's not really worse than it was ;)

Reply:
Well, I claim it's now broken for English too